src/tcp.c

```diff
@@ -103,14 +152,12 @@ enftun_tcp_connect(struct enftun_tcp* tcp,
             break;
     }
 
-    /*
```

Comment:
It's my fault this is broken into two commits but this removes a line added in dev commit `58e8782`